debian/changelog

```diff
@@ -1,4 +1,4 @@
-swtpm (0.5.0-1) RELEASED; urgency=medium
+swtpm (0.5.0) RELEASED; urgency=medium
```

Comment:
Can you please add your Signed-off-by: to the patch description?

And if you could make this a PR against the stable-0.5 branch that would help as well. Thank you!